test/api/fragments.test.js

```diff
@@ -135,6 +137,38 @@ describe('api/fragments', () => {
     });
   });
 
+  describe('postFragment', () => {
+    it('return a Promise', () => {
+      expect(postFragment(GET_FRAGMENT_OK)).toBeInstanceOf(Promise);
+    });
+
+    it('makes the correct request with settings body', () => {
+      postFragment(GET_FRAGMENT_OK);
+      expect(makeRequest).toHaveBeenCalledWith(expect.objectContaining({
+        method: METHODS.POST,
+        uri: '/api/fragments',
+        body: { code: `${GET_FRAGMENT_OK.code}`, guiCode: `${GET_FRAGMENT_OK.guiCode}` },
+        useAuthentication: true,
+      }));
+    });
+  });
+
+  describe('putFragment', () => {
+    it('return a Promise', () => {
+      expect(putFragment(GET_FRAGMENT_OK)).toBeInstanceOf(Promise);
+    });
+
+    it('makes the correct request with settings body', () => {
+      putFragment(GET_FRAGMENT_OK);
+      expect(makeRequest).toHaveBeenCalledWith(expect.objectContaining({
+        method: METHODS.PUT,
+        uri: `/api/fragments/${GET_FRAGMENT_OK.code}`,
+        body: { code: `${GET_FRAGMENT_OK.code}`, guiCode: `${GET_FRAGMENT_OK.guiCode}` },
```

Comment:
Why not `body: { code: GET_FRAGMENT_OK.code, guiCode: GET_FRAGMENT_OK.guiCode },`?